### PR TITLE
feat: implement sign-in mutation

### PR DESCRIPTION
**Pull request by denistakeda**
issue #4

### CODE REVIEW

File: server/db/mongo/controllers/users.js [hunk 1]
```diff
@@ -4,14 +4,19 @@ import {then} from '../../../utils/invokers';
 import jwt from 'jsonwebtoken';
 import {SECRET_KEY} from '../../../../config/env';
 
+const generateToken = user => ({
+  user,
+  token: jwt.sign(user._id, SECRET_KEY)
+});
+
 export const create = R.pipe(
```
Comment (sv3k) [inline server/db/mongo/controllers/users.js]: I would rename `create` to `signUp` for the sake of consistency.

Comment (sv3k) [reply server/db/mongo/controllers/users.js]: Also, is there a way to make this create/signUp method and signIn method look similar?
Currently `create` method has `(_, args)` inside `R.pipe`, but  `signIn` — vice versa.

Comment (denistakeda) [reply server/db/mongo/controllers/users.js]: Fixed

**Comment by sv3k** [comment]
👍

**Pull request merged**